### ISSUE TITLE
Run independent models in parallel first, even if other models are not independent

### ISSUE THREAD

**Issue by VEZY** [VEZY/PlantSimEngine.jl#40]
Related to #21 

At the time when one model is not runable in parallel for an object, we run every model in sequential mode, even the ones that are completely independent. 

Some of these completely independent models could be runable in parallel for time-steps or objects. These ones should be computed first in parallel, marked as simulated, and then we would compute the others in sequential mode.

Note that this is of **low priority** as I'm not sure will have much use-cases, if any. But it would be nice to have conceptually.

**Comment by Samuel-amap** [comment]
Closing this, somewhat outdated. Will refer to it in the more recent related issue though.
